### ISSUE TITLE
python3-datalad dependency unmet

### ISSUE THREAD

**Issue by adelavega** [datalad/datalad#2823]
#### What is the problem?
I'm guessing this is related to the latest emergency release, but I can't install datalad using apt

#### What steps will reproduce the problem?
```
apt-get update && apt-get install datalad
 >>The following packages have unmet dependencies:
 >>datalad : Depends: python3-datalad (= 0.10.3-1~nd90+1) but it is not going to be installed
```

Any tips on how to install a previous version? I'm guessing this will get resolved on its own soon once the channels are fully updated?

**Comment by yarikoptic-gitmate** [comment]
[GitMate.io](https://gitmate.io) thinks possibly related issues are https://github.com/datalad/datalad/issues/154 (Support python3), https://github.com/datalad/datalad/issues/1193 (datalad mv), https://github.com/datalad/datalad/issues/2693 (datalad aliases), https://github.com/datalad/datalad/issues/227 (datalad get  barfes), and https://github.com/datalad/datalad/issues/562 (datalad create   booms).

**Comment by adelavega** [comment]
Cool thanks, I'll test this out tomorrow. I worked something out in the meantime, just need it for a neuroscout docker image release

**Comment by yarikoptic** [comment]
solved with 0.10.3.1-2
